doc/user/content/sql/functions/to_char.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "to_char Function"
+description: "Converts a timestamp into a string using the specified format."
+menu:
+  main:
+    parent: 'sql-functions'
+---
+
+`to_char` returns a string created using the specified format string.
+
+The format string can be composed of any number of format tokens, interspersed with
+regular text. You can place a format token inside of double-quotes to emit it literally.
+
+## Examples
+
+**RFC 2822 Format**
+
+```sql
+SELECT to_char(TIMESTAMPTZ '2019-11-26 15:56:46 +00:00', 'Dy, Mon DD YYYY HH24:MI:SS +0000') AS formatted
+```
+```nofmt
+             formatted
+ ---------------------------------
+  Tue, Nov 26 2019 15:56:46 +0000
+```
+
+**Additional non-interpreted Text**
+
+```sql
+SELECT to_char(TIMESTAMPTZ '2019-11-26 15:56:46 +00:00', '"Welcome" to Mon, YYYY') AS formatted
+```
+```nofmt
+       formatted
+ ----------------------
+  Welcome to Nov, 2019
+```
+
+**Ordinal Modifiers**
+
+```sql
+SELECT to_char(TIMESTAMPTZ '2019-11-1 15:56:46 +00:00', 'Dth of Mon') AS formatted
+```
+```nofmt
+  formatted
+ ------------
+  6th of Nov
+```
+
+## Format Specifiers
+
+Regular specifiers:
+
+| Pattern                     | Description                                                                                      |
```

Comment:
"Pattern" here another name for these things. Seems like having one consistent name would make this nominally easier to reference.